src/main/java/uk/gov/hmcts/cmc/claimstore/events/paidinfull/PaidInFullCitizenNotificationHandler.java

```diff
@@ -33,14 +38,25 @@ public void onPaidInFullEvent(PaidInFullEvent event) {
     public void notifyDefendantForPaidInFull(PaidInFullEvent event) {
         Claim claim = event.getClaim();
         Map<String, String> parameters = aggregateParams(claim);
-        notificationService.sendMail(
-            claim.getDefendantEmail(),
-            notificationsProperties.getTemplates().getEmail().getClaimantSaysDefendantHasPaidInFull(),
-            parameters,
-            referenceForDefendant(claim.getReferenceNumber())
+
+        getDefendantEmail(claim).ifPresent(defendantEmail ->
+            notificationService.sendMail(
+                defendantEmail,
+                notificationsProperties.getTemplates().getEmail().getClaimantSaysDefendantHasPaidInFull(),
+                parameters,
+                referenceForDefendant(claim.getReferenceNumber())
+            )
         );
     }
 
+    private Optional<String> getDefendantEmail(Claim claim) {
+        if (StringUtils.isNotBlank(claim.getDefendantEmail())) {
+            return Optional.ofNullable(claim.getDefendantEmail());
```

Comment:
At this point it's guaranteed not null, so use `Optional.of(claim.getDefendantEmail())`

Reply:
Cheers